include/libzfs.h

```diff
@@ -520,7 +520,7 @@ extern nvlist_t *zfs_get_clones_nvl(zfs_handle_t *);
  */
```

Comment:
In header 'inheritted' -> 'inherited'

Reply:
not sure what you mean here....

Reply:
I started a review, but then I found nothing to add beyond the typo in commit message:
```
* Receiving an unencrypted stream as encrypted without
  needing to create a "dummy" encrypted parent so that
  encryption can be inheritted.
```
so it ended up being more picky than I intended, so feel free to ignore it :)